packages/gatsby-theme-docs/gatsby-browser.js

```diff
@@ -31,6 +31,8 @@ export const onClientEntry = (
         whitelistUrls: ['docs.commercetools.com', 'now.sh'],
       });
     });
+    if (window && !window.location.host.includes('.commercetools.com'))
+      document.body.classList.add('not-on-cookie-domain');
```

Comment:
Ah I guess this won't work because the banner is rendered after... 🤔

Let me see if I can find a better solution. I just would like to avoid using class names.

Reply:
Yes, that stupid script is not completed on load, it first triggers fetching a jquery somewhere and when that is there etc. pp.  
JS is imperative and hence fragile to what happens when on the time scale.  CSS is declarative and you can decide to just not care.   

We  _could_   do an engineering beauty contest and  hook a MutationObsever to the body element, watching for "childlist" changes and then manually killing / hiding them if they are the optanon stuff etc. pp.  

This beast is challenging the stubborn engineer in me, too that there _must_ be a clean solution but the manager in me says we're already 600% over what the topic should be worth in dev time budget :-)